src/state/modals/login/actions.ts

```diff
@@ -0,0 +1,12 @@
+import OmniAural from "omniaural"
+
+const modalsLoginHide = () => {
+  OmniAural.state.modals.login.show.set(false)
+}
+
+const modalsLoginShow = () => {
+  OmniAural.state.modals.login.show.set(true)
+}
+
+OmniAural.addActions(modalsLoginHide)
+OmniAural.addActions(modalsLoginShow)
```

Comment:
Not that it hurts but you can add both functions in one function call. Just to have less function calls to Omni
`OmniAural.addActions(modalsLoginHide, modalsLoginShow)`

Reply:
Ok nice. I actually tried `OmniAural.addActions([...,...])` before but it failed. I can just add each parameter though.